x-pack/plugins/licensing/common/license_update.ts

```diff
@@ -25,13 +27,24 @@ export function createLicenseUpdate(
   triggerRefresh$: Observable<unknown>,
   stop$: Observable<unknown>,
   fetcher: () => Promise<ILicense>,
+  maxRetryDelay: number,
   initialValues?: ILicense
 ) {
   const manuallyRefresh$ = new Subject<void>();
 
   const fetched$ = merge(triggerRefresh$, manuallyRefresh$).pipe(
     takeUntil(stop$),
-    exhaustMap(fetcher),
+    exhaustMap(() =>
+      defer(() => fetcher()).pipe(
+        retry({
+          delay: (_, retryCount) =>
+            maxRetryDelay > Math.pow(2, retryCount - 1) * 1000
+              ? timer(Math.pow(2, retryCount - 1) * 1000) 
+              : EMPTY,
```

Comment:
nit: Perform the math operation just once and reuse.

```suggestion
          delay: (_, retryCount) => {
            const nextDelay = Math.pow(2, retryCount - 1) * 1000;
            return maxRetryDelay > nextDelay ? timer(nextDelay) : EMPTY;
          },
```